src/styles/_variables.scss

```diff
@@ -1,9 +1,9 @@
 @use '../../projects/igniteui-angular/src/lib/core/styles/themes' as *;
 
-$palette: $light-material-palette;
-$schema: $light-material-schema;
-$typeface: $material-typeface;
-$type-scale: $material-type-scale;
+$palette: $light-fluent-palette;
+$schema: $light-fluent-schema;
+$typeface: $fluent-typeface;
+$type-scale: $fluent-type-scale;
 
```

Comment:
You don't need to commit this change.

Reply:
I agree with Silvia, this should be reverted back to material.